website/util/__init__.py

```diff
@@ -28,6 +29,30 @@
     'create_folder': 'file',
 }
 
+# Function courtesty of @brianjgeiger, moved from API utils
+def rapply(collection, func, *args, **kwargs):
+    """Recursively apply a function to all values in a dict or list
+
+    :param dict | list collection:
+    :param function func:
+    """
+    if isinstance(collection, collections.Mapping):
+        return {
+            key: rapply(value, func, *args, **kwargs)
+            for key, value in collection.iteritems()
+        }
+    elif isinstance(collection, list):
```

Comment:
`isinstance` can take a tuple as its 2nd argument, so you can get rid of the branch below:

```python
lif isinstance(collection, (list, tuple)):
```